.oxlintrc.json

```diff
@@ -0,0 +1,257 @@
+{
+    "$schema": "./node_modules/oxlint/configuration_schema.json",
+    "plugins": ["typescript", "unicorn", "react", "react-perf", "jest", "promise", "jsdoc", "oxc"],
+    "env": {
+        "builtin": true
+    },
+    "ignorePatterns": [
+        "node_modules",
```

Comment:
i wonder if this needs to be `**/node_modules` to get all the products folders too

(although it's all super fast so 🤷 )